### PR TITLE
Update MLX setup instructions for KF 1.5

### PR DESCRIPTION
**Pull request by ckadner**
Update the MLX installation instructions for Kubeflow 1.5

Tested Kubernetes providers:
* IBM Cloud Kubernetes service
* IBM Cloud OpenShift
* IBM Fyre OpenShift

Related:
* https://github.com/IBM/manifests/pull/41
* https://github.com/IBM/manifests/pull/44

/cc @Tomcli @yhwang @jbusche @rafvasq

### CODE REVIEW

File: docs/mlx-setup.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Deploy MLX on an existing Kubernetes cluster
 
 ## Prerequisites
+
 * An existing Kubernetes cluster:
    - **Min version: 1.17**
-   - **Max version: 1.21** 
+   - **Max version: 1.23**
```
Comment (Tomcli) [inline docs/mlx-setup.md:7]: with datashim, max is still 1.21. For 1.23 we need to wait for the next datashim release

File: docs/mlx-setup.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Deploy MLX on an existing Kubernetes cluster
 
 ## Prerequisites
+
 * An existing Kubernetes cluster:
    - **Min version: 1.17**
```
[the remark this answers is not in the document]
Comment (ckadner) [reply docs/mlx-setup.md]: ✅

**Comment by ckadner** [comment]
Thanks @Tomcli -- I updated the supported Kubernetes version

**Pull request merged**